android/sdk/src/main/java/org/jitsi/meet/sdk/BaseReactView.java

```diff
@@ -194,29 +194,10 @@ public void onWindowFocusChanged(boolean hasFocus) {
         super.onWindowFocusChanged(hasFocus);
 
         // https://github.com/mockingbot/react-native-immersive#restore-immersive-state
-
-        // FIXME The singleton pattern employed by RNImmersiveModule is not
-        // advisable because a react-native mobule is consumable only after its
-        // BaseJavaModule#initialize() has completed and here we have no
-        // knowledge of whether the precondition is really met.
         RNImmersiveModule immersive = RNImmersiveModule.getInstance();
 
         if (hasFocus && immersive != null) {
-            try {
-                immersive.emitImmersiveStateChangeEvent();
-            } catch (RuntimeException re) {
```

Comment:
Are we not worried about this anymore?

Reply:
Yes, the module itself now checks for it. It even does so in a cleaner way, by checking of there is a Catalyst instance attached to the React context: https://github.com/mockingbot/react-native-immersive/blob/698e7a31ca423fe9b7aff125cc8ea6f1f47b121f/android/src/main/java/com/rnimmersive/RNImmersiveModule.java#L73

Reply:
Ok, cool! Please fix the conflict then, and I'm happy to approve